index.d.ts

```diff
@@ -0,0 +1,90 @@
+export type Options = {
+	/**
+  Favor accuracy over speed.
+
+  @default false
+  */
```

Comment:
Incorrect indentation on all of these.

Reply:
I guessed you favor tabs by looking at https://github.com/imagemin/imagemin-pngquant/blob/main/index.d.ts. Think it should be fine now. Would be nice to have prettier (or alternative) to format automatically.